src/components/CheckedLabel/CheckedLabel.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {forwardRef, PropsWithChildren, ReactNode} from 'react';
+import {DOMProps, QAProps} from '../types';
+import {block} from '../utils/cn';
+
+import './CheckedLabel.scss';
+
+type Props = PropsWithChildren<{
```

Comment:
Maybe text should added by property label/text

Reply:
I think passing through `children` is more robust